feat/shiftStartTime

rasimoner opened the pull request:
- add shiftStartTime props
- create allMinutes with this prop.

[PT-262](https://pusulayazilimprojects.atlassian.net/browse/PT-262)

rasimonerr committed:
feat/shiftStartTime
- add shiftStartTime props
- create allMinutes with this prop.

(https://pusulayazilimprojects.atlassian.net/browse/PT-262)[PT-262]

rasimonerr committed:
feat/shiftStartTime
- add shiftStartTime props
- create allMinutes with this prop.

(https://pusulayazilimprojects.atlassian.net/browse/PT-262)[PT-262]

sabasayer commented:
ShiftStartTime is misleading because this calendar can be used in any project. Rename it something like slotStarTime or minutesRenderStartTime and add an explanation to the Readme.md about it.

rasimonerr committed:
feat/shiftStartTime
- The shiftStart property has been renamed to minuteRenderStartTime.
- The createAllMinutes request parameters have been converted to an object.
- formatted code with prettier

(https://pusulayazilimprojects.atlassian.net/browse/PT-262)[PT-262]

rasimoner commented:
> ShiftStartTime is misleading because this calendar can be used in any project. Rename it something like slotStarTime or minutesRenderStartTime and add an explanation to the Readme.md about it.

i changed ShiftStartTime to minutesRenderStartTime, thanks for your advice.

rasimonerr committed:
feat/shiftStartTime
- the pusula/calendar package version number increased

(https://pusulayazilimprojects.atlassian.net/browse/PT-262)[PT-262]
